### PR TITLE
Bug fix and test class for new label

### PR DESCRIPTION
**Pull request by hanjukim**
- updateColor should be called after updating quads on alignText (If don't, opacity won't applied)
- New and old label comparison (with bounding box)

### CODE REVIEW

File: samples/Cpp/TestCpp/Classes/LabelTest/LabelTestNew.cpp [hunk 1]
```diff
@@ -1312,3 +1313,63 @@ void LabelCharMapTest::step(float dt)
 {
     return "Not crash and show [Test123] when using unknown character.";
 }
+
+LabelTTFOldNew::LabelTTFOldNew()
+{
+    auto s = Director::getInstance()->getWinSize();
+    float delta = s.height/4;
+
+    auto label1 = CCLabelTTF::create("Cocos2d-x Label Test", "arial", 24);
```
Comment (dumganhar) [inline samples/Cpp/TestCpp/Classes/LabelTest/LabelTestNew.cpp]: Please remove CC prefix.

File: samples/Cpp/TestCpp/Classes/LabelTest/LabelTestNew.cpp [hunk 2]
```diff
@@ -1312,3 +1313,63 @@ void LabelCharMapTest::step(float dt)
...
+    addChild(label1, 0, kTagBitmapAtlas1);
+    label1->setAnchorPoint(CCPoint(0.5f, 0.5f));
+    label1->setPosition(CCPoint(s.width/2, delta * 2));
```
Comment (dumganhar) [inline samples/Cpp/TestCpp/Classes/LabelTest/LabelTestNew.cpp]: `CCPoint` --> `Point`

Comment (hanjukim) [inline samples/Cpp/TestCpp/Classes/LabelTest/LabelTestNew.cpp]: Thank you for notifying me. Done.

**Comment by dumganhar** [comment]
Thanks.

**Comment by WenhaiLin** [comment]
@hanjukim  fixed y position for new label at: https://github.com/cocos2d/cocos2d-x/pull/5197.

**Comment by hanjukim** [comment]
@Dhilan007 It works great! Thank you.

**Pull request merged**